packages/core/src/data-editor/data-editors-fns.ts

```diff
@@ -173,6 +173,25 @@ export function copyToClipboard(cells: readonly (readonly GridCell[])[], columnI
         return str;
     }
 
+    const formatBoolean = (val: boolean | null | undefined): string => {
+        switch (val) {
+            case true:
+                return "TRUE";
+
+            case false:
+                return "FALSE";
+
+            case undefined:
+                return "INDETERMINATE";
+
+            case null:
+                return "EMPTY";
```

Comment:
I would argue null should format as FALSE or an empty string.

Reply:
Hhmm, yeah perhaps `""`.